yt/visualization/fixed_resolution.py

```diff
@@ -97,6 +97,15 @@ class FixedResolutionBuffer:
         ("index", "theta"),
         ("index", "dtheta"),
     )
+    antialias = traitlets.Bool(True)
+    buff_size = traitlets.Tuple(traitlets.Int(), traitlets.Int())
+    period = traitlets.Tuple(YTDimensionfulTrait(dimensions = dims.length),
+                             YTDimensionfulTrait(dimensions = dims.length))
+    periodic = traitlets.Bool(True)
+    bounds = traitlets.List(YTDimensionfulTrait(dimensions = dims.length))
+    axis = traitlets.Enum([0,1,2,3,4])
```

Comment:
headsup: I see a collision here with https://github.com/yt-project/yt/pull/3827/commits/b02df62bc936f15a3b66bf3adcede95265b075d9 from #3827

Reply:
thanks, I'm sure this will get lots of changes in the meantime